[jules] Gestalt Swarm CLI Compilation Fix

## 🎯 Objetivo
Fix compilation error in Gestalt Swarm CLI.

## 📋 Problema
El agente de Swarm CLI diagnosticó el problema: "Swarm CLI compilation failed with exit code 1: error[E0433]: failed to determine the explicit crate type target for gestalt_swarm"

El problema es:
1. crate type multiplicado: gestalt_swarm tiene múltiples bin targets que compilan mal
2. bin duplicado: hay múltiples binaries definidos para swarm

## 📁 Archivos con Problemas
- `gestalt_swarm/Cargo.toml` — crate-type issue
- `gestalt_swarm/src/main.rs` — CLI entry point

## ✅ Tareas Requeridas

### 1. Diagnóstico
```bash
cd E:\scripts-python\gestalt-rust
cargo build -p gestalt_swarm 2>&1 | Select-String -Pattern "error" | Select-Object -First 20
```

### 2. Fix Cargo.toml
Revisar gestalt_swarm/Cargo.toml y asegurar:
- Solo un `[[bin]]` target
- `crate-type` correcto (bin vs lib vs cdylib)
- Dependencies correctas

### 3. Verificar main.rs
- Entry point correcto
- Imports válidos
- No referencias a módulos que no existen

### 4. Build completo
```bash
cargo build --release 2>&1 | Select-String -Pattern "Finished|error" | Select-Object -First 10
```

### 5. Test CLI
```bash
cargo run -p gestalt_swarm -- --help 2>&1
```

## 📊 Criterios de Éxito
- [ ] `cargo build -p gestalt_swarm` compila sin errores
- [ ] `cargo build --release` compila todo sin errores
- [ ] `gestalt_swarm --help` muestra help

## 🔧 Entorno
- **Repo:** iberi22/gestalt-rust
- **Rama:** main
- **Stack:** Rust

## 📝 Notas
- MCP Bridge ya está integrado (commit b65a41c)
- Cortex Memory backend ya funciona (commit b65a41c)
- Solo falta el Swarm CLI

--- a/Cargo.toml
+++ b/Cargo.toml
@@ -7,6 +7,7 @@ members = [
     "gestalt_timeline", "gestalt_mcp", "gestalt_ui",
     "gestaltctl",
     "synapse-agentic",
+    "gestalt_swarm",
 ]
 resolver = "2"
 

--- a/gestalt_swarm/Cargo.toml
+++ b/gestalt_swarm/Cargo.toml
@@ -1,0 +1,21 @@
+[package]
+name = "gestalt_swarm"
+version = "1.0.0"
+edition = "2021"
+authors = ["Gestalt Team"]
+description = "Gestalt Swarm CLI for multi-agent coordination"
+
+[[bin]]
+name = "swarm"
+path = "src/main.rs"
+
+[dependencies]
+clap = { version = "4.5", features = ["derive"] }
+tokio = { version = "1.49.0", features = ["full"] }
+anyhow = "1.0"
+serde = { version = "1.0", features = ["derive"] }
+serde_json = "1.0"
+tracing = "0.1"
+tracing-subscriber = { version = "0.3", features = ["env-filter"] }
+gestalt_core = { path = "../gestalt_core" }
+synapse-agentic = { path = "../synapse-agentic" }

--- a/gestalt_swarm/src/main.rs
+++ b/gestalt_swarm/src/main.rs
@@ -1,0 +1,54 @@
+use clap::{Parser, Subcommand};
+use tracing::info;
+use tracing_subscriber::{fmt, prelude::*, EnvFilter};
+
+#[derive(Parser, Debug)]
+#[command(name = "swarm")]
+#[command(about = "Gestalt Swarm CLI", long_about = None)]
+struct Cli {
+    #[command(subcommand)]
+    command: Option<Commands>,
+
+    /// Verbose output
+    #[arg(short, long, global = true)]
+    verbose: bool,
+}
+
+#[derive(Subcommand, Debug)]
+enum Commands {
+    /// Check the status of the swarm
+    Status,
+    /// Run a swarm task
+    Run {
+        /// The goal for the swarm
+        #[arg(short, long)]
+        goal: String,
+    },
+}
+
+#[tokio::main]
+async fn main() -> anyhow::Result<()> {
+    let args = Cli::parse();
+
+    let level = if args.verbose { "debug" } else { "info" };
+    tracing_subscriber::registry()
+        .with(fmt::layer())
+        .with(EnvFilter::try_from_default_env().unwrap_or_else(|_| EnvFilter::new(level)))
+        .init();
+
+    match args.command {
+        Some(Commands::Status) => {
+            println!("🐝 Gestalt Swarm is active and ready.");
+        }
+        Some(Commands::Run { goal }) => {
+            info!("Running swarm task: {}", goal);
+            println!("🚀 Executing goal: {}", goal);
+            // Swarm orchestration logic would go here
+        }
+        None => {
+            println!("Gestalt Swarm CLI. Use --help for available commands.");
+        }
+    }
+
+    Ok(())
+}

